apps/test/unit/templates/sectionProgressV2/ProgressFeedbackBannerTest.jsx

```diff
@@ -0,0 +1,144 @@
+import {fireEvent, render, screen, waitFor} from '@testing-library/react';
+import React from 'react';
+import sinon from 'sinon';
+
+import {UnconnectedProgressFeedbackBanner} from '@cdo/apps/templates/sectionProgressV2/ProgressFeedbackBanner';
+import i18n from '@cdo/locale';
+
+import {expect} from '../../../util/reconfiguredChai';
+
+describe('UnconnectedProgressFeedbackBanner', () => {
+  const fakeFetch = sinon.spy();
+  const fakeCreate = sinon.spy();
```

Comment:
optional nit: I prefer to have an empty function (`() => {}`) as the default and override the default whenever we need to add the spy.

Reply:
Got it... this makes sense - it helps it stay more self contained.  I was trying to avoid adding lines of code into individual tests by keeping this as the default.  That being said, I still declared the spies outside the individual test - I think this was the right move to be able to still use the `afterEach`.  Open to feedback on this choice though.